app/views/items/_form.html.erb

```diff
@@ -43,6 +43,14 @@
                 <%= f.input_field :package_size, class: "form-control" %>
               <% end %>
 
+              <% if Flipper.enabled?(:enable_packs) %>
```

Comment:
This is a very ugly solution, I'd love suggestions on how we can clean this up

Reply:
I think this is fine overall, hard to do this without some calculation. You could move the `selected_request_units` into a view helper.

```
def selected_item_request_units(item)
  item_request_unit_names = @item.persisted? ? @item.request_units.pluck(:name) : []
  current_organization.request_units.select { |unit| item_request_unit_names.include?(unit.name)}.pluck(:id) %>
end
```

```suggestion
              <% if Flipper.enabled?(:enable_packs) %>
                <%= f.input :request_units, label: "Additional Custom Request Units" do %>

                  <%= f.association :request_units, as: :check_boxes, collection: current_organization.request_units, checked: selected_item_request_units(@item), label_method: :name, value_method: :id, class: "form-check-input" %>
                <% end %>
              <% end %>
```